packages/shared/src/contracts/registry.ts

```diff
@@ -13,7 +13,7 @@ const POLYGON: Contracts = {
 
 const MUMBAI: Contracts = {
   SUBSCRIPTION_MANAGER: '0xb9015d7b35ce7c81dde38ef7136baa3b1044f313',
-  COORDINATOR: '0x0f019Ade1D34399D946CF2f161386362655Dd1A4',
+  COORDINATOR: '0x8E49989F9D3aD89c8ab0de21FbA2E00C67ca872F',
```

Comment:
Opened https://github.com/nucypher/nucypher-ts/issues/310. The bulk of this module should  be externalized to the NPM package. See also 
https://github.com/nucypher/nucypher-contracts/issues/124 for on-going discussion